coders/heic.c

```diff
@@ -73,6 +73,7 @@
 #include "MagickCore/string-private.h"
 #include "MagickCore/module.h"
 #include "MagickCore/utility.h"
+#include "MagickCore/attribute.h"
```

Comment:
We have the includes in alphabetical order 😁

Reply:
Don't need this one since using alpha_trait